package.json

```diff
@@ -12,6 +12,11 @@
     "e2e:publish": "./scripts/e2e-publish.sh",
     "e2e:resolutions": "node ./scripts/e2e-resolutions.js",
     "examples": "npm run examples --workspaces --if-present",
+    "lint": "npm run lint --workspaces --if-present",
+    "lint:fix": "npm run lint:fix --workspaces --if-present",
+    "test": "npm run test --workspaces --if-present",
+    "test:node": "npm run test:node --workspaces --if-present",
```

Comment:
I'm fully in support of these but should they be part of this PR?  If so, can we also update devp2p so it has a `test:node` script as an alias of `test`?

Reply:
I would pledge yes, this is such a tiny side change not affecting anything apart of having a bit more comfort. I will add the alias.